Resources/Prototypes/Entities/Markers/Spawners/mobs.yml

```diff
@@ -29,6 +29,32 @@
     prototypes:
       - MobCorgiIan
 
+- type: entity
+  name: Lia Spawner
+  id: SpawnMobLia
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: green
+        - state: ai
+    - type: ConditionalSpawner
+      prototypes:
+        - MobCarpLia
+
+- type: entity
+  name: Lia Spawner
+  id: SpawnMobLia
```

Comment:
these both have the same id and i don't for the life of me know why the linter isn't throwing an error on this.